src/containers/Profile/ProfileCategoriesBar/index.js

```diff
@@ -0,0 +1,3 @@
+import ProfileCategoriesBar from './ProfileCategoriesBar';
```

Comment:
This file just imports and exports another module we can simply import the module as it is just one module

Reply:
I use the _[Component Folder Pattern](https://medium.com/styled-components/component-folder-pattern-ee42df37ec68)_. It helps make code more searchable. However if the convention is against this, I will revert.